crest/Assets/Crest/Crest/Scripts/Crest.asmdef

```diff
@@ -1,3 +1,21 @@
-﻿{
-	"name": "Crest"
-}
+{
+    "name": "Crest",
+    "references": [
+        "GUID:d60799ab2a985554ea1a39cd38695018"
+    ],
+    "includePlatforms": [],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": false,
+    "precompiledReferences": [],
+    "autoReferenced": true,
+    "defineConstraints": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.postprocessing",
+            "expression": "",
+            "define": "UNITY_POSTPROCESSING_ENABLED"
```

Comment:
is this intentional? just checking

Reply:
The define? It is.

Reply:
do we want to add the postprocessing package?

Reply:
All this does is create the define symbol if the post-processing package is in the project. It doesn't add any packages.

See: https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html#define-symbols

Reply:
Ok so adding it to the references list above does not add it to the project?

Reply:
Thumbs up for merging anyway

Reply:
Correct.